src/OSPSuite.R/OSPSuite.R.csproj

```diff
@@ -27,6 +27,8 @@
     <ProjectReference Include="..\OSPSuite.Infrastructure.Autofac\OSPSuite.Infrastructure.Autofac.csproj" />
     <ProjectReference Include="..\OSPSuite.Infrastructure.Import\OSPSuite.Infrastructure.Import.csproj" />
     <ProjectReference Include="..\OSPSuite.Infrastructure\OSPSuite.Infrastructure.csproj" />
+    <ProjectReference Include="..\OSPSuite.Presentation\OSPSuite.Presentation.csproj" />
+    <ProjectReference Include="..\OSPSuite.UI\OSPSuite.UI.csproj" />
```

Comment:
We can't add these. That means that the UI and presentation project are required for R but that should not be the case. R should not need access to any of that.